_data/simple-icons.json

```diff
@@ -2430,6 +2430,11 @@
             "hex": "525DDC",
             "source": "https://roots.io/"
         },
+        {
+            "title": "Royal Dutch Shell",
```

Comment:
Not something that necessarily requires a change, just want to point out that just "Shell" would be fine, too. From [its Wikipedia page](https://en.wikipedia.org/wiki/Royal_Dutch_Shell):

> Royal Dutch Shell [...], commonly known as Shell [...]

Reply:
I like that! I was thinking about just Shell too, as I would always refer to it as just Shell, but unsure because the Wikipedia article and Google used the full name. But I think as the full name is rarely used, just Shell is the better choice.